src/control/cmd/daos_server/storage.go

```diff
@@ -107,13 +107,26 @@ func (cmd *storagePrepareCmd) Execute(args []string) error {
 		}
 	}
 
-	if cmd.ScmOnly || !cmd.NvmeOnly {
-		// Prepare SCM modules to be presented as pmem kernel devices
+	for cmd.ScmOnly || !cmd.NvmeOnly {
```

Comment:
I still don't understand why this is a for loop...

Reply:
because you can't break out of an if, I've changed it back for an if

Reply:
Ah, got it. Seems like that suggests that the code in this block should be in its own function that can return early, then.